test/jasmine/tests/download_test.js

```diff
@@ -45,11 +40,11 @@ describe('Plotly.downloadImage', function() {
 
     it('should create link, remove link, accept options', function(done) {
         downloadTest(gd, 'jpeg', done);
-    });
+    }, LONG_TIMEOUT_INTERVAL);
```

Comment:
See https://github.com/jasmine/jasmine/commit/68ba5b6d48624c063b4eeae1bdd63c5aa84a3177 for example.